README.md

```diff
@@ -0,0 +1,47 @@
+# puppeteer-har-server
+
+This project exposes functionality offered by [`puppeteer-har`][0] over an HTTP interface. With it, you can start recording HTTP traffic on a running instance of Chrome and retrieve that HTTP traffic as a HTTP archive (`.har`).
```

Comment:
> retrieve that HTTP traffic as a HTTP archive

Getting back to my roots as a pedantic "code" reviewer, I think this should be "*an* HTTP archive".